### PR TITLE
This is a sampe pull request 

### PR DESCRIPTION
**Pull request by jeremyweber72**
asdasda

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/jeremyweber72/laughing-octo-robot/pull/2" target="_blank">ReviewNinja

File: foo.txt [hunk 1]
```diff
@@ -1 +1 @@
-foo next4
```
Comment (jeremyweber72) [inline foo.txt]: !resolved  !completed

Comment (jeremyweber72) [inline foo.txt]: THis doesnt seem to be right

File: foot.txt [hunk 2]
```diff
@@ -1 +1,4 @@
 foo next3 
+bar
+123
+some text
```
Comment (jeremyweber72) [inline foot.txt]: Last line

File: foot.txt [hunk 3]
```diff
@@ -1 +1,4 @@
 foo next3 
+bar
+123
```
Comment (jeremyweber72) [inline foot.txt]: Next to Last line

Comment (jeremyweber72) [inline foot.txt]: This should not be here,

File: foot.txt [hunk 4]
```diff
@@ -1 +1,4 @@
 foo next3
```
Comment (jeremyweber72) [inline foot.txt]: test

**Pull request merged**